traject_config.rb

```diff
@@ -0,0 +1,104 @@
+
+require 'traject/macros/marc_format_classifier'
+require 'traject/macros/marc21_semantics'
+require 'solrizer'
+require_relative 'app/utils/identifier'
+extend Traject::Macros::Marc21Semantics
+extend Traject::Macros::MarcFormats
+
+def ark_from_alexandria_uri(uri)
+  md = /http:\/\/alexandria\.ucsb\.edu\/lib\/(ark:\/\d{5}\/.*)/.match(uri)
+  md[1] if md
+end
+
+settings do
+  # provide "writer_class_name", "Traject::JsonWriter"
+  # TODO use the solr.yml
+  provide "solr.url", "http://localhost:8983/solr/development"
```

Comment:
Not really, we don't have a Rails environment loaded.  You can pass a solr url (to override this default) on the command line and I'll add that to the documentation.